courses/views.py

```diff
@@ -76,14 +76,16 @@ def get_context_data(self, *, object_list=None, **kwargs):
 class CourseView(DetailView):
     """Course view"""
 
-    queryset = Course.objects.prefetch_related("courserun_set").all()
+    model = Course
     template_name = "course_detail.html"
-
+    
     def get_context_data(self, **kwargs):
         return {
             **super().get_context_data(**kwargs),
             **get_js_settings_context(self.request),
             "user": self.request.user,
+            "courseware_object": self.object,
+            "type": "course"
```

Comment:
why `type` is added here ?

Reply:
we will use it later. just added.

Reply:
we can add this latter then.

Reply:
update tests for view change if exists, add if not.

Reply:
test already there and we are just updating the course_detail template with context.